src/pages/about/index.js

```diff
@@ -1,19 +1,118 @@
+import Leaders from './_leaders'
+import { getLocationHash } from 'common/utility'
 import Layout from 'components/layout/layout'
-import { SEO } from 'components/containers'
+import { localize, Localize, WithIntl } from 'components/localization'
+import { Container, Wrapper, Flex, SEO } from 'components/containers'
+import { Header, Text, Image } from 'components/elements'
+import device from 'themes/device'
 
-const About = () => {
-    const [navigation, setNavigation] = useState('story')
+const Background = styled.div`
+    background: var(--color-black);
+    width: 100%;
+    height: 100%;
+`
+
+const StyledContainer = styled(Container)`
+    padding: 12rem 19rem;
+    white-space: nowrap;
+
+    @media ${device.laptopL} {
+        padding: 12rem 4rem;
+    }
+    @media ${device.tablet} {
+        padding: 8rem 4rem;
+    }
+    @media ${device.mobileL} {
+        padding: 8rem 2rem;
+    }
+`
+const ContentWrapper = styled.div`
+    margin-top: ${props => props.margin_top || 'none'};
+    white-space: normal;
+    max-width: 79.2rem;
+`
+
+const LeadershipWrapper = styled(Flex)`
+    @media ${device.tabletL} {
+        flex-direction: column;
+
+        ${Wrapper} {
+            width: 28.2rem;
+            margin-bottom: 4rem;
+        }
+        ${Header} {
+            text-align: center;
+        }
+        ${Text} {
+            text-align: center;
+        }
+    }
+`
+
+const LeadershipTitle = styled(Header)`
+    margin-bottom: 0.8rem;
+`
+
+const LeadershipPosition = styled(Header)`
+    margin-bottom: 1.6rem;
+`
 
+const NavigationWrapper = styled(Flex)`
+    margin-top: 1.6rem;
+`
+
+const Navigation = styled(Flex)`
+    cursor: pointer;
+`
+
+const Separator = styled.span`
+    width: 2px;
+    height: 3rem;
+    background: white;
+`
+
+const StyledHeader = styled(Header)`
+    transition: color 0.25s;
+    ${props =>
+        props.active
+            ? css`
+                  color: var(--color-white);
+              `
+            : css`
+                  color: rgba(255, 255, 255, 0.32);
+                  &:hover {
+                      color: rgba(255, 255, 255, 0.5);
+                  }
+              `}
+`
+const TrailNavigation = styled.span`
+    height: 4px;
+    width: 4.6rem;
+    background: ${props => (props.active ? 'var(--color-red)' : 'var(--color-black)')};
+    margin: 1rem 0;
+    transition: background 0.25s;
+`
+function useTabState(tab) {
+    const [active_tab, setActiveTab] = useState(tab)
+    const setTab = tab => {
+        if (tab === active_tab) return
+        setActiveTab(tab)
+        navigate(`#${tab}`)
+    }
+    return [active_tab, setTab]
+}
+const About = () => {
+    const [active_tab, setTab] = useTabState('story')
+    const is_story = active_tab === 'story'
+    const is_leadership = active_tab === 'leadership'
+    useEffect(() => {
+        const new_tab = getLocationHash() || 'story'
+        setTab(new_tab)
+    })
```

Comment:
I think we only want to this to fire on mount?

```suggestion
    }, [])
```

Reply:
oops didn't mean to approve it!

Reply:
actually nevermind, you can change it back to the old way as in this way the component is not re-rendered when you update the locationhash from the link in the footer.